publications/joss/paper.md

```diff
@@ -113,15 +114,15 @@ outlined below and visualized in following Figure:
   
 - **Predictor module**: Implements the following tasks to predict atmospheric parameters: 
   - Perform Scale/normalize processes on the observational fluxes
-  - Deploys the trained CNNs model 
+  - Deploy the trained CNNs model 
   - Predict atmospheric parameters, i.e., effective temperature, gravity, carbon-to-oxygen ratio, and metallicity 
-  - Visualizes the processed observational dataset and the uncertainty in the predicted results
+  - Visualize the processed observational dataset and the uncertainty in the predicted results
   
 - **StatVisAnalyzer module**: Provides a set of functions to perform the following tasks: 
-  - Explores and processes the synthetic datasets
-  - Performs the chi-square test to evaluate the similarity between two datasets
-  - Calculates confidence intervals and standard errors
-  - Functions t
+  - Explore and processes the synthetic datasets
+  - Perform the chi-square test to evaluate the similarity between two datasets
+  - Calculate confidence intervals and standard errors
+  - Function t(???)
```

Comment:
This seems to be a fragment, so more information should be added or the bullet point should be removed.